openlabcmd/openlabcmd/node.py

```diff
@@ -46,9 +48,9 @@ def update(self, update_dict):
     def from_zk_bytes(cls, zk_bytes):
         node_dict = json.loads(zk_bytes[0].decode('utf8'))
         # ctime and mtime is millisecond in zk.
-        ctime, mtime = zk_bytes[1].ctime, zk_bytes[1].mtime
+        ctime, mtime = zk_bytes[1].ctime / 1000, zk_bytes[1].mtime / 1000
```

Comment:
seems, you can use the `iso8601` to do this format converting, it is more convenient

Reply:
I just convert the `int` type millisecond to second here . Can't see how iso8601 works here? Suggestion?